.github/workflows/release.yml

```
@@ -15,7 +15,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)